static/manifest.json

```diff
@@ -59,6 +59,7 @@
   "minimum_chrome_version": "88",
   "name": "Rainbow DEVELOPMENT BUILD",
   "permissions": [
+    "downloads",
```

Comment:
This might be an issue because it could require an ugly permission approval on the next update; something to ask Bruno about.

Reply:
@brunobar79 any thoughts here?

Reply:
moving away from permission changes by doing this:
```
  const data = await getWallets();
  const csvContent = generateCSV(data as WalletObj[], walletNames);
  const blob = new Blob([csvContent], { type: 'text/csv' });
  const url = URL.createObjectURL(blob);

  // Create a temporary anchor to initiate the download
  const tempLink = document.createElement('a');
  tempLink.href = url;
  tempLink.download = 'rainbow_addresses.csv';
  document.body.appendChild(tempLink);
  tempLink.click();
  document.body.removeChild(tempLink);

  // Release the object URL after use
  URL.revokeObjectURL(url);
  ```